public/toolkit/styles/molecules/_tabs.scss

```diff
@@ -2,11 +2,10 @@
 // Tabs
 //
 
-.tabs dd, 
+.tabs dd,
```

Comment:
do you know additional elements `.tabs dd` selects? does this just select on tab-titles anyways? If it's only selecting on tab-titles the selector on line 5 can be removed and the width styles below could be pulled up